config/config-feature-flags.yaml

```diff
@@ -39,7 +39,8 @@ data:
   # and only allows one pipelinerun to run on a node at a time.
   # Setting it to "disabled" will not apply any coschedule policy.
   #
-  # TODO: add links to documentation and migration strategy
+  # See more in the workspace documentation about Affinity Assistant
+  # https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#affinity-assistant-and-specifying-workspace-order-in-a-pipeline
```

Comment:
can you also link to affinity assistant docs here? https://github.com/tektoncd/pipeline/blob/main/docs/additional-configs.md#customizing-the-pipelines-controller-behavior

Reply:
Sure, links added

Reply:
sorry, I meant link to the new affinity assistant docs from https://github.com/tektoncd/pipeline/blob/main/docs/additional-configs.md#customizing-the-pipelines-controller-behavior, not the other way around.

Reply:
Oh sorry, just fixed